.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: ruby
+rvm:
+- 2.4.2
+
+before_script:
+ - chmod +x ./script/cibuild.sh
+
+script: ./script/cibuild.sh
+
+deploy:
+  - provider: firebase
+    edge: true
+    skip_cleanup: true
+    project: malagadevfest-5f472
+    token: $FIREBASE_TOKEN
```

Comment:
@Azahar @zolastro we'll need to update the FIREBASE_TOKEN environment var in your travis-ci.org project